### ISSUE TITLE
Need clarification on Windows setup

### ISSUE THREAD

**Issue by MetadataDeluxe** [MuseosAbiertos/csv2exif#8]
I had trouble getting mme to run on Windows 10. It would be helpful to add more details to the download/setup instructions.

These are the components I think are necessary, but I might have done something wrong along the way. This is just a rough list that needs expert editing.

- MME https://github.com/MuseosAbiertos/Museum-Metadata-Embedder
  - Clone the repository or download the .zip file to the folder you want to work in and rename it "mme"
- Exiftool	exiftool.org
  - download the zip for windows, unzip and copy exiftool(-k).exe inside the "mme" folder and rename it exiftool.exe
- Perl https://www.activestate.com/products/perl/
  - download and place in the same directory as the "mme" folder.
- Python 3 https://www.python.org/downloads/
  - [I got an error after installing from https://www.python.org/downloads/. After installing from the Microsoft Store, it worked. Need to work out the best method and best installation location for Python to work]
- colorama https://pypi.org/project/colorama/
  - Command line: pip install colorama
- pysimplegui https://www.pysimplegui.org/en/latest/
  - Command line: pip install pysimplegui

**Comment by hvdwolf** [comment]
I agree. We need better explanation for all OSes. It was already there but upon renewal of the site, it partly disappeared.

In the mean time you can try this  "one in all" exe: "Museum-Metadata-Embedder-0.3-win-x86_64.zip". 
I uploaded this one plus an AppImage and Debian deb to mega.nz again. See this [mme](https://mega.nz/folder/tcUGCCIL#A8QusBugffMDC5JRrIYp5Q) folder.
Already create on 10-11 July, but I wanted to add this to a release on this Github when I had the MacOS bundle ready, but as always Apple behaves differently.
And last week was, despite holidays, very hectic.

**Comment by hvdwolf** [comment]
Coming back to your first post: Why install Perl? 
Exiftool.exe is a perl compiled (compressed) executable. You don't need perl for exiftool.

**Comment by MetadataDeluxe** [comment]
The  "one in all" exe: "Museum-Metadata-Embedder-0.3-win-x86_64.zip" works very well, just a simple double click to run!

About perl... I was following the instructions on https://docs.museosabiertos.org/en/museum-metadata-embedder/download
You're right, I removed perl and MME ran. That's good, one less thing to install.

**Comment by hvdwolf** [comment]
> The "one in all" exe:

😀 I only now see it in your exactly quoted reply. I meant to write "all in one".

But good to hear it works fine.

**Comment by MetadataDeluxe** [comment]
One for all, all for one!

**Comment by mrtngrsbch** [comment]
> The "one in all" exe: "Museum-Metadata-Embedder-0.3-win-x86_64.zip" works very well, just a simple double click to run!
> 
> About perl... I was following the instructions on https://docs.museosabiertos.org/en/museum-metadata-embedder/download You're right, I removed perl and MME ran. That's good, one less thing to install.

I apologize if my documentation can be confusing and requires even more work. I see that at this point I should split it into Linux/MacOS and Windows. Normally I do it first in Spanish and then translate it, but in our case I think it is not the best idea.

On the other hand, I decided to separate the extensive documentation in my documentation site (https://docs.museosabiertos.org/), where I also have the jExifToolGUI documentation in Spanish. This is also done by @hvdwolf with jExifToolGUI and I think it is NOT the best collaborative tool to maintain the manuals. Right now I wonder if it is not better to make a GitHub Pages for a better collaborative work.

I welcome suggestions

**Comment by mrtngrsbch** [comment]
PS: The 'readme.md' is now in Spanish because I needed them for local public, but I think it doesn't make sense anymore. Today I'll put them in English.

**Comment by hvdwolf** [comment]
What you could do is make the English one the main one, and have in the top of it a line/link saying "For a Spanish version click here" leading to a README_es.md.
If you do locally a "git mv README.md README_es.md" you have already that one arranged. Then you create a new README.md in English with the link inside it.

**Comment by mrtngrsbch** [comment]
I will follow your instructions. The English version is now online as the only version. The ES version is on my documentation site.
My intention to have documentation on my site is to have extended and task related documentation.

**Comment by MetadataDeluxe** [comment]
@mrtngrsbch don't apologize, this is a complex project and the documentation is very good, it just needs fine tuning.

**Comment by hvdwolf** [comment]
I fully agree. You already did a great job.

**Comment by mrtngrsbch** [comment]
> These are the components I think are necessary, but I might have done something wrong along the way. This is just a rough list that needs expert editing.

Yes, you are right and I will create a page dedicated only to the installation.
